packages/manager/apps/web/client/app/hosting/user-logs/translations/Messages_fr_FR.json

```diff
@@ -19,5 +19,6 @@
   "hosting_user_logs_dashboard_users_description": "Vous pouvez créer de nouveaux utilisateurs si vous désirez ouvrir l'accès à l'interface OVHcloud Web Statistics à d'autres collaborateurs que ceux spécifiés comme contacts principaux (administratif, technique et facturation). Ces comptes seront administrables dans le tableau ci-dessous :",
   "hosting_user_logs_table_popover_modify": "Modifier",
   "hosting_user_logs_table_popover_update_password": "Changer le mot de passe",
-  "hosting_user_logs_table_popover_delete": "Supprimer"
+  "hosting_user_logs_table_popover_delete": "Supprimer",
```

Comment:
Why English text in French locale file? looks like the entair file is not translated.

Reply:
Which file/translation are you referring too?

From this file `packages/manager/apps/web/client/app/hosting/user-logs/translations/Messages_fr_FR.json`, I can't find any English content.